src/engine/source/schemf/test/src/component/schema_test.cpp

```diff
@@ -141,3 +149,18 @@ INSTANTIATE_TEST_SUITE_P(SchemaTest,
                                            LoadTuple(R"({"a": {}})", false),
                                            LoadTuple(R"({"a": [{"type": "text"}]})", false),
                                            LoadTuple(R"([{"a": {"type": "keyword"}}])", false)));
+
+TEST(SchemaTest, ArrayItem)
```

Comment:
Is this a particular case? If it is just a case, could you document the test?

Reply:
I think the test is self documented. It just tests array items get and exists.